elasticapm/traces.py

```diff
@@ -642,6 +642,26 @@ def set_transaction_result(result, override=True):
         transaction.result = result
 
 
+def get_transaction_id():
+    """
+    Returns the current transaction ID
+    """
+    transaction = execution_context.get_transaction()
+    if not transaction:
+        return
+    return transaction.id
+
+
+def get_transaction_trace_parent_id():
```

Comment:
I think this could just be called `get_trace_id`? Transaction is implied, since you can't have a trace without a transaction; and it's still useful to get the trace ID when you're in the context of a span.

Reply:
Yeah, most of this naming is trying to be true to the underlying objects. I noticed traces don't have an ID directly, it's the `trace_parent.id`, thus the name. I'll rename them if `trace_parent.id` is considered the trace ID.

Reply:
Done.